Firmware/messages.c

```diff
@@ -8,150 +8,150 @@
 #include "Configuration_prusa.h"
 
 //internationalized messages
-const char MSG_AUTO_HOME[] PROGMEM_I1 = ISTR("Auto home"); ////
-const char MSG_BABYSTEP_Z[] PROGMEM_I1 = ISTR("Live adjust Z"); //// c=18
+const char MSG_AUTO_HOME[] PROGMEM_I1 = ISTR("Auto home"); ////c=18
+const char MSG_BABYSTEP_Z[] PROGMEM_I1 = ISTR("Live adjust Z"); ////c=18
 const char MSG_BABYSTEP_Z_NOT_SET[] PROGMEM_I1 = ISTR("Distance between tip of the nozzle and the bed surface has not been set yet. Please follow the manual, chapter First steps, section First layer calibration."); ////c=20 r=12
-const char MSG_BED[] PROGMEM_I1 = ISTR("Bed"); ////
-const char MSG_BED_DONE[] PROGMEM_I1 = ISTR("Bed done"); ////
-const char MSG_BED_HEATING[] PROGMEM_I1 = ISTR("Bed Heating"); ////
-const char MSG_BED_LEVELING_FAILED_POINT_LOW[] PROGMEM_I1 = ISTR("Bed leveling failed. Sensor didnt trigger. Debris on nozzle? Waiting for reset."); ////c=20 r=5
+const char MSG_BED[] PROGMEM_I1 = ISTR("Bed"); ////c=13
+const char MSG_BED_DONE[] PROGMEM_I1 = ISTR("Bed done"); ////c=20
+const char MSG_BED_HEATING[] PROGMEM_I1 = ISTR("Bed Heating"); ////c=20
+const char MSG_BED_LEVELING_FAILED_POINT_LOW[] PROGMEM_I1 = ISTR("Bed leveling failed. Sensor didn't trigger. Debris on nozzle? Waiting for reset."); ////c=20 r=6
```

Comment:
have you tested this change? Does it change message formatting?

Reply:
> Please verify if all reported \x00 texts are valid, some of them really look like untranslated for some reason (e.g. "Printer IP Addr" in CZ)

There are some and also found few others that have been missing. Question is if we should add these?